docs/cpp/customize-cpp-settings-cpp.md

```diff
@@ -133,18 +147,18 @@ Note, this is an example of all fields. You do not need to specify all fields in
 ## Configuration properties
 
 - `name`
-  A friendly name that identifies a configuration. `Linux`, `Mac`, and `Win32` are special identifiers for configurations that will be autoselected on those platforms. The status bar in VS Code will show you which configuration is active. You can also select the label in the status bar to change the active configuration.
+  A friendly name that identifies a configuration. `Linux`, `Mac`, and `Win32` are special identifiers for configurations that are autoselected on those platforms. The status bar in VS Code shows you which configuration is active. You can also select the label in the status bar to change the active configuration.
 
 - `compilerPath` (optional)
```

Comment:
Can you remove the "(optional)" string?  While it is optional, so are most other properties. The difference being that this is one we *really* want people to set.